core/src/Instance/AadAuthority.cs

```diff
@@ -35,63 +35,51 @@ namespace Microsoft.Identity.Core.Instance
 {
     internal class AadAuthority : Authority
     {
-        private const string AadInstanceDiscoveryEndpoint = "https://login.microsoftonline.com/common/discovery/instance";
-        private static readonly HashSet<string> TrustedHostList = new HashSet<string>()
+        internal static readonly HashSet<string> TrustedHostList = new HashSet<string>()
```

Comment:
I assume this should be read-only, consider using a property { get; private set;} and maybe use an ImmutableHashSet

Reply:
yes, it is read only, did not managed to add ImmutableHashSet - it is not visible due to some issues with multi targeting projects